### PR TITLE
October blog post

### PR DESCRIPTION
**Pull request by olenagerasimova**

### CODE REVIEW

File: _posts/2022/10/2022-10-31-release-notes.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+keywords:
+  - artipie
+  - binary artifacts
+  - hosting of artifacts
+  - Abstract storage
+  - rest api
+---
+
+Artipie was represented in [DevOops](https://devoops.ru/) conference on October 19 in 
+Saint-Petersburg. Our current team-leader Denis Gurus told how to Artipie with Maven and Docker
```
Comment (dgarus) [inline _posts/2022/10/2022-10-31-release-notes.md]: `ho to` -> `how to use` ?

File: _posts/2022/10/2022-10-31-release-notes.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+repositories. Find the workshop announce [here](https://devoops.ru/talks/827cdd91b1fb40cbae91d1335dcd54d1/).
+
+[v0.27.4](https://github.com/artipie/artipie/releases/tag/v0.27.4) was released on October 27. The 
+last and current months we are working on Artipie monitoring and metrics. The goal is to provide easy to
+use and understand system, which will allow us to know what is happening in the Artipie, how fast
+the artifacts can be uploaded and downloaded, catch and fix any problems right on time. The firsts
```
Comment (dgarus) [inline _posts/2022/10/2022-10-31-release-notes.md]: `The firsts` - > `The first` ?

**Review by dgarus — CHANGES_REQUESTED** [review]

**Comment by olenagerasimova** [comment]
@dgarus thanks, corrected

**Review by dgarus — APPROVED** [review]
@olenagerasimova thanks!

**Pull request merged**